conda_build/license_family.py

```diff
@@ -14,11 +13,11 @@
 GPL
 BSD
 MIT
-APACHE
+Apache
 PSF
-PUBLICDOMAIN
-PROPRIETARY
-OTHER
+Public-Domain
```

Comment:
Actually, this (and `'Other'` --> `'OTHER'`, etc.) break *all* recipes using these license families... The check tests
```Python
if remove_special_characters(normalize(license_family)) not in allowed_license_families
```
The function `normalize` changes the license family specified in the recipe to use "ALL CAPS", which therefore will *never* match `Public-Domain`, `Other`, `Apache`, or `Proprietary`.